test/test_reference.py

```diff
@@ -0,0 +1,132 @@
+"""Test Robust Reference"""
+import pytest
+
+import numpy as np
+import mne
+import random
+from pyprep.reference import robust_reference, remove_reference
+from pyprep.noisy import Noisydata
+
+
+def make_reference_mne_object(sfreq=1000., t_secs=200, n_freq_comps=5,
```

Comment:
This can be put into a "conftest.py" file, and then imported using pytest.markfixture(). Please read up on how to do this, and then I can show you in person if you're having trouble.